tools/dm/query-status.md

```diff
@@ -0,0 +1,224 @@
+                                ],
+                                "firstPos": "(bin|000001.000001, 3130)",        # sharding DDL 语句起始位置。
+                                "synced": [                                     # 被 `Sync` 处理单元读取了已执行 sharding DDL 语句的上游分表。
+                                    "`test`.`t2`"
+                                    "`test`.`t3`"
+                                    "`test`.`t1`"
+                                ],
+                                "unsynced": [                                   # 未执行该 sharding DDL 语句的上游表。如有上游表未完成同步，`blockingDDLs` 为空。
+                                ]
+                            }
+                        ],
+                        "synced": false         # 增量同步是否已和上游一致，并与上游数据库拥有相同的 binlog 位置。由于后台 `Sync` 单元并不会实时刷新保存点，当前值为 "false"
+                                                # 并不一定代表发生了同步延迟。
+                    }
+                }
+            ],
+            "relayStatus": {    # The synchronization status of the relay log.
+                "masterBinlog": "(bin.000001, 3234)",                               # 上游数据库的 binlog 位置。
+                "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-14",    # 上游数据库的 binlog GTID 信息。
+                "relaySubDir": "c0149e17-dff1-11e8-b6a8-0242ac110004.000001",       # 当前使用的 relay log 子目录。
+                "relayBinlog": "(bin.000001, 3234)",                                # 已被拉取至本地存储的 binlog 位置。
+                "relayBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-14",     # 已被拉取至本地存储的 binlog GTID 信息。
+                "relayCatchUpMaster": true,     # 本地 relay log 同步进度是否与上游一致。
+                "stage": "Running",             # relay log 的 `Sync` 处理单元状态
+                "result": null
+            }
+        },
+        {
+            "result": true,
+            "worker": "172.17.0.3:10081",
+            "msg": "",
+            "subTaskStatus": [
+                {
+                    "name": "test",
+                    "stage": "Running",
+                    "unit": "Load",
+                    "result": null,
+                    "unresolvedDDLLockID": "",
+                    "load": {                   # `Load` 处理单元的同步信息。
+                        "finishedBytes": "115", # 已加载字节数。
+                        "totalBytes": "452",    # 待加载字节数。
+                        "progress": "25.44 %"   # 加载进度。
+                    }
+                }
+            ],
+            "relayStatus": {
+                "masterBinlog": "(bin.000001, 28507)",
+                "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
+                "relaySubDir": "c0149e17-dff1-11e8-b6a8-0242ac110004.000001",
+                "relayBinlog": "(bin.000001, 28507)",
+                "relayBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
+                "relayCatchUpMaster": true,
+                "stage": "Running",
+                "result": null
+            }
+        },
+        {
```

Comment:
we can remove L101 ~ L129, duplicated with L72 ~ L100.
original CN doc's fault.